### ISSUE TITLE
8. Conversation and Text Generation - oritenting

### ISSUE THREAD

**Issue by JunsolKim** [UChicago-Computational-Content-Analysis/Readings-Responses-2023#15]
Post questions here for this week's oritenting readings: Goldberg, Amir, Sameer B. Srivastiva, V. Govind Manian, William Monroe and Christopher Potts. 2016. “Fitting In or Standing Out? The Tradeoffs of Structural and Cultural Embeddedness”. American 
Sociological Review 81(6): 1190-1222.

**Comment by konratp** [comment]
The authors mention that the case selection is rather limiting the potential to make any generalizable claim about the issue. Corporate culture in particular strikes me as a difficult thing for scholars to to analyze, as corporations generally portray themselves in a better light. Are there other ways of textual analysis that don't rely on individual companies providing private email communication to assess such questions? One thing that comes to mind are glassdoor or indeed reviews, though those themselves might provide a more negative image of the companies as the reviews are written by people who have left the company.

**Comment by Sirius2713** [comment]
This paper use "a mid-sized technology firm served as our research site".  Is it possible to conclude different results if the setting is different? I understand it's very difficult to get internal email data of a company, not to mention expand the experiment to a large scale. But adding on Konratp's question, are there any other ways to do a cross-sectional experiment without relying on internal email data?

**Comment by Jiayu-Kang** [comment]
I love this topic and really like how the authors analyze the data from both the structural and cultural dimensions! I'm wondering how we can generalize the results to other forms of communication besides emails. This may be achieved simply by replicating the same research method to data such as meeting transcripts - those data are even more difficult to acquire though. It might also be interesting to see whether the results also hold true for organizations with different structures/sizes, or from different industries/social context etc.

**Comment by GabeNicholson** [comment]
> This paper use "a mid-sized technology firm served as our research site". Is it possible to conclude different results if the setting is different? I understand it's very difficult to get internal email data of a company, not to mention expand the experiment to a large scale. But adding on Konratp's question, are there any other ways to do a cross-sectional experiment without relying on internal email data?



> 

I think that's a good hypothesis to test out. I would assume that smaller organizations would favor people who stand out while larger ones follow a different culture. I also would assume smaller organizations to be much different from each other and larger organizations to be culturally more similar to each other.

**Comment by Hongkai040** [comment]
> > This paper use "a mid-sized technology firm served as our research site". Is it possible to conclude different results if the setting is different? I understand it's very difficult to get internal email data of a company, not to mention expand the experiment to a large scale. But adding on Konratp's question, are there any other ways to do a cross-sectional experiment without relying on internal email data?
> 
> > 
> 
> I think that's a good hypothesis to test out. I would assume that smaller organizations would favor people who stand out while larger ones follow a different culture. I also would assume smaller organizations to be much different from each other and larger organizations to be culturally more similar to each other.

I kind of agree with this hypothesis. But what I am thinking is does organization's culture play some important roles in the favor of brokers or nonconformists? When I was reading this paper, I was thinking of Apple, Steve Jobs, and the slogan "Think different".  Maybe an organization's culture could serve as meta culture (I don't know, just created this term :)) that influences their preferences.

**Comment by ValAlvernUChic** [comment]
> > > This paper use "a mid-sized technology firm served as our research site". Is it possible to conclude different results if the setting is different? I understand it's very difficult to get internal email data of a company, not to mention expand the experiment to a large scale. But adding on Konratp's question, are there any other ways to do a cross-sectional experiment without relying on internal email data?
> > 
> > 
> > > 
> > 
> > 
> > I think that's a good hypothesis to test out. I would assume that smaller organizations would favor people who stand out while larger ones follow a different culture. I also would assume smaller organizations to be much different from each other and larger organizations to be culturally more similar to each other.
> 
> I kind of agree with this hypothesis. But what I am thinking is does organization's culture play some important roles in the favor of brokers or nonconformists? When I was reading this paper, I was thinking of Apple, Steve Jobs, and the slogan "Think different". Maybe an organization's culture could serve as meta culture (I don't know, just created this term :)) that influences their preferences.

To add, I wonder what this would look like in organizations that are necessarily more hierarchical and where network constraints and language used become a function of such a hierarchy - for example, being in a uniformed organization (the army, police etc.). In addition, in Singapore at least, it is rare that someone in the army/police is involuntary released from the service (unless committing a crime etc.). Instead, they lose a rank or even worse for some, are kept at the same rank for the rest of their careers. In this case, I imagine extended time kept at one rank could be a form of negative attainment.

**Comment by isaduan** [comment]
The authors 'conjecture that  findings are not limited to organizations or to professional attainment' ... they 'anticipate that the contingent advantages of structural and cultural embeddedness should play out similarly in social settings that are not strictly organizational.' What are some other playgrounds to test the hypothesis?

**Comment by pranathiiyer** [comment]
Adding to Isabella's comment, given that "professional jargon" can sometimes tend to be nuanced, how can we ensure that even this could semantically fit into the predefined categories of cognitive, semantic, and emotional categories that the authors talk about?

**Comment by NaiyuJ** [comment]
While reading this article, I'm thinking of the relationship between cultural norms and linguistic norms. The authors seem to assume that people who fit in culturally learned to understand and match the linguistic norms followed by their colleagues. Then from the linguistic norms, we can know the cultural norms. This logic makes more sense in the context of oral communications. But for written communications in the workplace, is it necessary that linguistic norms can efficiently relate to cultural norms? 

Other than that, I'm thinking of the distinctions between informative language and uninformative language used in the emails in terms of shaping the cultural norms. Theoretically, we should focus only on informative language, but in practice, uninformative language may be more influential on the algrithom?

**Comment by Jasmine97Huang** [comment]
The authors noted that "cultural fit measures the extent to which the semantic categories in her outgoing messages correspond to the categories in her incoming messages." Such measurement assumes the cultural fit equals cultural harmony. However, a culture of diversity that welcomes challenges would be deem as culturally unfit even when each member demonstrates coherent pattern of divergence. How is such culture addressed in the study?

**Comment by hsinkengling** [comment]
This work is pretty interesting. One question I had, given our topic this week, is whether it's feasible to go beyond regression and really leverage the conversational data to ask more granular questions about the dynamics of cultural and structural embeddedness, or would that kind of research be too controversial due to privacy concerns?

For example, perhaps we can classify emails into different types of interactions using the tools from week 5 and draw out some network graphs to plot the interaction between different actors.

**Comment by sizhenf** [comment]
Very interesting read! I wonder if the same mode could be used in studying mobility/immigration. For eg, how structural and cultural fit can affect one's achievement when moving to a new country.

**Comment by YileC928** [comment]
My concern is mainly about robustness and generalizability. Is email communication a data source representative enough to operationalize cultural and structural embeddedness? It is possible that employees who are close to each other (e.g., in the same office or geographically close) tend to interact through in-person meetings or phone calls; email may not be very informative in reflecting culture. Perhaps geographical distance could serve as a control variable to capture such effect and surveys could be done to validate network structure.

**Comment by hshi420** [comment]
I was wondering about the performance of the methods in other kinds of corpora. I think the distribution of variables in different corpora can vary a lot, which can affect the resutls.

**Comment by chentian418** [comment]
I have a question about measuring cultural embeddedness. The authors use a predefined set of cognitive, semantic, and emotional categories to code each e-mail. I was wondering if we could generate the set of words in each category using a BERT model and pre-train the historical emails. Then we might get a more customized and precise measurement of cultural embeddedness.

**Comment by LuZhang0128** [comment]
This paper remains me of Becker's 1974 paper that argues art is a collective action. People do want to be both conventional and make small innovations. I'd be more than happy to use the approach proposed in this paper and empirically study the paper proposed in Becker's paper again.

**Comment by mikepackard415** [comment]
I like the theoretical and computational approach of this paper quite a bit. I wonder if economists would be interested in replicating this across a wide range of firms of different sizes, in different industries, to study whether there are patterns that predict firm success. That would be pretty interesting, because it would give allow us to look at the evolution of strategies for success at multiple levels (individual and firm) simultaneously.

**Comment by Qiuyu-Li** [comment]
I think the paper is really interesting! I'm wondering why there's a network constraint. Why would people be willing to fitting into a group facing a network constraint? Because intuitively they should be good at talking with people and expanding their social network.

**Comment by Emily-fyeh** [comment]
I would also want to know how to examine the phenomenon in other social games. Also, I would be curious how this work can extend to a dynamic version capturing the shifting working/office culture, since micro-culture is always changing through different times and settings. (e.g. the pandemic)

**Comment by shosamane** [comment]
Business email communication is kept formal and (usually) bereft of emotional and cognitive cues. It is much different to spoken business communication. How accurate would it be to measure these cues from this kind of formal setting?

One might also change their cultural fit over time - one might change the characteristics of the outgoing to align with the incoming ones realising that is the norm at the workplace, thus making (by metrics defined) the person more culturally fit, although this might not be the case. How accurate are these kinds of observations?

**Comment by melody1126** [comment]
In this paper, the authors use LIWC to label emails as an operationalization of cultural fit in an organization : "The more e-mails she sends that exhibit different stylistic, topical, and emotional characteristics than the ones she receives, the lower her cultural fit." (pg. 1200-1) However, I am skeptical about whether email messages are an effective operationalization of the concept of cultural embeddedness in an organization. There are many more dimensions to cultural fit here, such as conversations by the water coolers about sports or other non-work related matters that demonstrate cultural fit as much, if not more, as business communications over emails.

**Comment by ttsujikawa** [comment]
I really liked the author's approach to social network topic in firm structure. While the author articulates individual culture fit by analyzing email communication, communication among employees have got much complicated with more dynamics and nestedness. Furthermore, COVID makes people move to virtual space with video meeting and such. I wonder how much we could apply this theory to today's real-world under such situation.
